ledger/ledger-api-common/src/main/scala/com/digitalasset/platform/server/api/ApiException.scala

```diff
@@ -3,9 +3,11 @@
 
 package com.daml.platform.server.api
 
-import io.grpc.{Status, StatusRuntimeException}
+import io.grpc.StatusRuntimeException
 
 import scala.util.control.NoStackTrace
 
 /** The sole purpose of this class is to give StatusRuntimeException with NoStacktrace a nice name in logs. */
-class ApiException(status: Status) extends StatusRuntimeException(status) with NoStackTrace
+class ApiException(exception: StatusRuntimeException)
```

Comment:
Weird code, as I haven't found any other way to convert it while preserving the trailers as well as keeping it `NoStackTrace`.